src/main/java/uk/gov/hmcts/darts/hearings/mapper/GetAnnotationsResponseMapper.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.darts.hearings.mapper;
+
+import lombok.experimental.UtilityClass;
+import uk.gov.hmcts.darts.common.entity.AnnotationDocumentEntity;
+import uk.gov.hmcts.darts.common.entity.AnnotationEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.hearings.model.Annotation;
+import uk.gov.hmcts.darts.hearings.model.AnnotationDocument;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.collections4.ListUtils.emptyIfNull;
+
+@UtilityClass
+@SuppressWarnings({"PMD.TooManyMethods"})
```

Comment:
Do we need this considering the class only has 5 methods?

Reply:
removed